File: Daily Challanges/2026-02-08.js
function calculatePenaltyDistance(rounds) {
  let distance = 0
  for(const rnd of rounds){
    distance += (5 - rnd) * 150
  }
  return distance;
}

const test1 = calculatePenaltyDistance([4, 4]);
const test2 = calculatePenaltyDistance([5, 5]);
const test3 = calculatePenaltyDistance([4, 5, 3, 5]);
const test4 = calculatePenaltyDistance([5, 4, 5, 5]);
const test5 = calculatePenaltyDistance([4, 3, 0, 3]);

const check1 = 300;
const check2 = 0;
const check3 = 450;
const check4 = 150;
const check5 = 1500;

console.log(`Test1: ${test1 === check1}`);
console.log(`Test2: ${test2 === check2}`);
console.log(`Test3: ${test3 === check3}`);
console.log(`Test4: ${test4 === check4}`);
console.log(`Test5: ${test5 === check5}`);

File: Daily Challanges/2026-02-February/2026-02-08.py
def calculate_penalty_distance(rounds):
    distance = 0
    for rnd in rounds:
        distance += (5 - rnd) * 150
    return distance

test1 = calculate_penalty_distance([4, 4])
test2 = calculate_penalty_distance([5, 5])
test3 = calculate_penalty_distance([4, 5, 3, 5])
test4 = calculate_penalty_distance([5, 4, 5, 5])
test5 = calculate_penalty_distance([4, 3, 0, 3])

check1 = 300
check2 = 0
check3 = 450
check4 = 150
check5 = 1500

print(f"Test1: {test1 == check1}")
print(f"Test2: {test2 == check2}")
print(f"Test3: {test3 == check3}")
print(f"Test4: {test4 == check4}")
print(f"Test5: {test5 == check5}")